### ISSUE TITLE
the output codewords

### ISSUE THREAD

**Issue by shellhuang1227** [marcottelab/akhmetov-2016#1]
Hello,
I ran the make_codewords.py but only 4 codewords (not 8) were generated, which were  ACGT, CTAG, GATC, TGCA. Did you meet this problem? Besides, the output codewords did not meet the biological parameters (i.e. GC content) if I change the length of codewords.  I will be appreciate it if you could  help me. Thank you very much.

**Comment by Superbest** [comment]
Thanks for making the issue! The codewords you have given (ACGT, CTAG, GATC and TGCA) all have 50% GC, so they do meet the parameters. Can you provide an example for the case that violates the condition?

Regarding the number of codewords, please see [comment at the top of that file](https://github.com/marcottelab/akhmetov-2016/blob/b47a19c9bfb2d68b9f0430646edc39be32171914/DNA%20codec/make_codewords.py#L5-L7).

**Comment by shellhuang1227** [comment]
Yeah, I ran it for codewords of length 6, and I got lots of codewords and some codewords were CGGCTAC, CGGTGGG, CGTAATC, CGTAGGA, CGTCACT, CGTGGTT, CGTTCCG. AGGTAGA, AGTATGT. Obviously, their GC contents do not meet the biological constraint.

**Comment by Superbest** [comment]
@shellhuang1227 Yes, I see the problem now - thank you! Looks like a bug, and I think I see where the problem is. Regrettably I am very busy the next few weeks (pull requests welcome!) but I think this is an easy fix. I'll take a look soon and see if I can take care of it!

**Comment by shellhuang1227** [comment]
Ok. Thank you very much.
